start.sh

```diff
@@ -28,5 +31,9 @@ if [ "$(ls -A /root/repoFiles/)" ]; then
   rm -rf tmp/
 fi
 
+echo 'starting ssh server...'
 # -D flag avoids executing sshd as a daemon
-/usr/sbin/sshd -D
+/usr/sbin/sshd -D &
+
+echo 'git server ready'
+wait $!
```

Comment:
Looking at the [`sshd` man page](https://linux.die.net/man/8/sshd), I think that what we want to do is:

```
/usr/sbin/sshd -D -e
```

where `-e` will cause `sshd` to log to stderr instead of some system log file. For me, on starting up, I would now see:

```
configuring authorized_keys...
creating repo...
Initialized empty Git repository in /home/git/repo/
starting ssh server...
Server listening on 0.0.0.0 port 22.
Server listening on :: port 22
```

which I think is fine for output, and can remove the `echo 'git server ready'` output and consequently the `wait $1` command as well.

Reply:
Good find. I'll apply this.